package.json

```diff
@@ -100,7 +100,7 @@
     "@types/glob": "^7.1.3",
     "@types/gulp": "4.0.8",
     "@types/inquirer": "^7.3.1",
-    "@types/jasmine": "^3.5.4",
+    "@types/jasmine": "^3.6.3",
```

Comment:
Can we have our `@types/jasmine` and  and `jasmine` versions match?

Reply:
Done, although there's nothing actually keeping them in sync on npm.

Reply:
Oh interesting, I was under the understanding they were  in lockstep.